src/_includes/header.html

```diff
@@ -85,3 +85,7 @@
       </div>
 </header>
 <!-- End Header -->
+
+{% if page.url == "/" or page.url == "/careers/" %}
+   {% include sm_cta_slice.html%}
```

Comment:
Is `sm` short for `software-modernisation`? Will that still be obvious in the future?

Reply:
It is yes, but we can obviously alter the naming to be more clearer